File: c++/Product.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string.h>
#include "Hardware.cpp"

using namespace std;

    class Product : public Hardware
    {
        private:
            //atribut
            int price;
            int idProduct;
        
        public:
            //konstruktor
            Product(){
            }

            //get dan set product

            void setPrice(int price){
                this->price = price;
            }

            int getPrice(){
                return this->price;
            }

            void setId_product(int idProduct){
                this->idProduct = idProduct;
            }

            int getId_product(){
                return this->idProduct;
            }

            //destructor
            ~Product(){
            }
            
    };

File: java/Product.java
public class Product extends Hardware{
    
    private int price;
    private int idProduct;

    Product(){
    }

    //get dan set Produk

    public void setPrice(int price){
        this.price = price;
    }

    public int getPrice(){
        return this.price;
    }

    public void setId_Product(int idProduct){
        this.idProduct = idProduct;
    }

    public int getId_Product(){
        return this.idProduct;
    }



}
